addons/hr_expense/static/src/views/expense_form_view.js

```diff
@@ -17,28 +17,25 @@ export class ExpenseFormController extends FormController {
         const t = this.env._t;
         const previousOnClickViewButton = this.env.onClickViewButton;
         useSubEnv({
-            async onClickViewButton({ clickParams, record}) {
-                if (clickParams.name === "action_submit_expenses") {
+            async onClickViewButton(params) {
+                const record = this.model.root;
+                if (params.clickParams.name === "action_submit_expenses") {
```

Comment:
you can merge these two nested ifs into one single if with an and between both conditions, since they have the same else clause